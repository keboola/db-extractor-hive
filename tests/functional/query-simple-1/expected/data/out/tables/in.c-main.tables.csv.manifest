{
    "destination": "in.c-main.tables",
    "incremental": false,
    "columns": [
        "tab_name"
    ],
    "column_metadata": {
        "tab_name": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.sourceName",
                "value": "tab_name"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "tab_name"
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            }
        ]
    }
}