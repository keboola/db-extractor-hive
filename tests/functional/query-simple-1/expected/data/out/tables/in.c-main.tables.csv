"chars"
"incremental"
"internal"
"sales"
"special_types"
